src/Storages/StorageView.h

```diff
@@ -47,20 +47,14 @@ class StorageView final : public IStorage
     static String replaceQueryParameterWithValue (const String & column_name, const NameToNameMap & parameter_values, const NameToNameMap & parameter_types);
     static String replaceValueWithQueryParameter (const String & column_name, const NameToNameMap & parameter_values);
 
-    void setParameterValues (NameToNameMap parameter_values_)
+    NameToNameMap getParameterTypes() const
```

Comment:
Not sure, but it looks like an unnecessary copy
```suggestion
    const NameToNameMap & getParameterTypes() const
```

Reply:
addressed it